### ISSUE TITLE
Unable to run with no config file

### ISSUE THREAD

**Issue by SHADOW-REX** [hyprwm/hyprpaper#193]
Hyprpaper used to work earlier without any config files purely through ipc, but with the latest updates it errors out:
`❯ hyprpaper &
[1] 1588
[LOG] Welcome to hyprpaper!                                                                                                                                                                                        
built from commit  ()
terminate called after throwing an instance of 'std::runtime_error'
  what():  Could not find config in HOME, XDG_CONFIG_HOME, XDG_CONFIG_DIRS or /etc/hypr.
[1]  + 1588 IOT instruction  hyprpaper`

**Comment by jjonojj** [comment]
+1, i got this issue too.

**Comment by Myphz** [comment]
Related to #192

**Comment by Marty42780** [comment]
It seems that creating an empty config file is enough to get Hyperpaper working again.

```bash
~ ❱ hyprpaper
[LOG] Welcome to hyprpaper!
built from commit  ()
terminate called after throwing an instance of 'std::runtime_error'
  what():  Could not find config in HOME, XDG_CONFIG_HOME, XDG_CONFIG_DIRS or /etc/hypr.
fish: Job 1, 'hyprpaper' terminated by signal SIGABRT (Abort)

~ | 134 ❱ touch ~/.config/hypr/hyprpaper.conf

~ ❱ hyprpaper
[LOG] Welcome to hyprpaper!
built from commit  ()
[WARN] Monitor eDP-1 does not have a target! A wallpaper will not be created.
[LOG] hyprpaper socket started at /run/user/1000/hypr/9a09eac79b85c846e3a865a9078a3f8ff65a9259_1724682760_20702455/.hyprpaper.sock (fd: 4)
```

As previously mentioned by @Myphz this seems to be a revert of https://github.com/hyprwm/hyprpaper/issues/87, is there any way to make the documentation aware of the change or restore the ability to not have a config file.

**Comment by SHADOW-REX** [comment]
> It seems that creating an empty config file is enough to get Hyperpaper working again.

You're right, and here i already updated my config to kill and restart hyprpaer on every wall change :)

**Comment by khaosdoctor** [comment]
This is still happening as of 2025, isn't there a base config that can be used instead of completely failing?

**Comment by Myphz** [comment]
> This is still happening as of 2025, isn't there a base config that can be used instead of completely failing?

Literally just an empty file
